### ISSUE TITLE
Use of Ti.UI.iOS is undefined on Android

### ISSUE THREAD

**Issue by parnanzio** [FokkeZB/ti-html2as#26]
I just updated the version of this module to the latest version (v 1.3.3): on iOS is working fine (simulator and device).
On Android there is a problem: `Cannot read property 'AttributedString' of undefined` raised by a call to `Ti.UI.iOS.AttributedString`.

```
[ERROR] TiExceptionHandler: (main) [0,16204] - In nl.fokkezb.html2as.js:1,69
[ERROR] TiExceptionHandler: (main) [0,16204] - Message: Uncaught TypeError: Cannot read property 'AttributedString' of undefined
[ERROR] TiExceptionHandler: (main) [0,16204] - Source: ,a]}))}return t}var l=e("htmlparser2"),r=e("entities"),s=([Ti.UI.iOS.Attribute
[ERROR] V8Exception: Exception occurred at nl.fokkezb.html2as.js:1: Uncaught TypeError: Cannot read property 'AttributedString' of undefined
```

SDK: 5.0.1
Module: 1.3.3
Device: Nexus 5 (5.1.1)

**Comment by parnanzio** [comment]
I just fixed it in my local file at line #6:

```
var ns = Ti.UI;
var references = [
  Ti.UI.AttributedString
];
if (parseInt(Ti.version.split('.')[0], 10) < 4){
	references = [
	  Ti.UI.iOS.AttributedString,
	  Ti.UI.AttributedString
	];
	ns = Ti.UI.iOS;	
}
```

**Comment by FokkeZB** [comment]
Thanks, I've updated the master
